.travis.yml

```diff
@@ -26,6 +26,12 @@ before_script:
   - sh -e /etc/init.d/xvfb start
   - sleep 3 # give xvfb some time to start
 
+before_install:
+  - npm install --no-save --no-optional --dev npm@5.8.0
```

Comment:
`Usage of the `--dev` option is deprecated. Use `--only=dev` instead.`
c.f. https://travis-ci.org/coala/gh-board/jobs/408440200#L458

Reply:
Actually it isnt necessary because of `--no-save`, but I have kept it in so that it is explicit which type of dependency it is in `package.json`